packages/walletkit/local.tar.gz

```diff
@@ -0,0 +1 @@
+No such object: contract_artifacts/local
```

Comment:
Did you mean to check this in? Looks like an artifact of a failed build or something?